tests/Fixer/Import/GlobalNamespaceImportFixerTest.php

```diff
@@ -1021,6 +1028,11 @@ function abc($foo, \Bar $bar = null) {}
                  * @return array<string, ?Bar<int, foo>>|null
                  */
                 function abc($foo, Bar $bar = null) {}
+
+                /**
+                 * @implements IteratorAggregate<Foo>
```

Comment:
This is not fixed as expected because [`GlobalNamespaceImportFixer::traverseDocBlockTypes()`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/bff36f9421d50407bf03cd4880205f484df9b29b/src/Fixer/Import/GlobalNamespaceImportFixer.php#L689-L725) only traverses [`Annotation::getTagsWithTypes()`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/bff36f9421d50407bf03cd4880205f484df9b29b/src/DocBlock/Annotation.php#L112-L120).

The question is, are you ok with expanding that list?

-

Reply:
They all are "tags with types", so I cannot see why not.